README.md

```diff
@@ -97,6 +111,21 @@ Gets cached item and moves it to the front
 const item = cache.get('myKey')
 ```
 
+## getMany
+
+### Method
+
+Gets multiple cached items and moves them to the front
```

Comment:
🟠 
@kibertoad 
> moves them to the front

Not sure if I understand the code correctly, but this might imply that if we pass `['a', 'b']`, then they'll be moved to the front as `a`, `b`, whereas they'll in fact be moved to the front as `b`, `a`? 🤔

Reply:
It's an LRU cache, meaning that whenever we run out of space, we start evicting elements that were used last. so getMany will bump all of the keys that were retrieved. Do you feel that description might be improved to make that clearer?

Reply:
adjusted the docs, because yeah, that was phrased in a confusing way